src/types.ts

```diff
@@ -75,35 +75,53 @@ type SectionType = 'autocomplete' | 'recommendations' | 'custom';
 
 export type SectionConfiguration = {
   type?: SectionType;
-  identifier: string;
   displayName?: string;
   numResults?: number;
   // This property will only take effect when using the component and not the hook
   displaySearchTermHighlights?: boolean;
   ref?: React.RefObject<HTMLElement>;
 };
 
-export interface AutocompleteSection extends SectionConfiguration {
+export interface AutocompleteSectionConfiguration extends SectionConfiguration {
   type?: 'autocomplete';
-  data: Item[];
+  indexSection: string;
+  /** @deprecated use indexSection field instead */
+  identifier?: string;
 }
 
-export interface RecommendationsSection extends SectionConfiguration {
+export interface RecommendationsSectionConfiguration extends SectionConfiguration {
   type: 'recommendations';
-  data: Item[];
+  indexSection: string;
+  podId: string;
   itemIds?: string[];
```

Comment:
This is old but I don't believe it's accurate. We don't use itemIds as part of the section configuration. We attach the items from the data information [here](https://github.com/Constructor-io/constructorio-ui-autocomplete/blob/b3305afa3d696abfc58578bfac874aaef70e0604/src/utils.ts#L209) for `trackRecommendationView`

Do you think `itemIds?: string[];` has any value here or smth I might be missing?

Reply:
I believe `term` and `itemIds` are parameters for the `getRecommendations` calls not for tracking